### DISCUSSION TITLE
Ayuda: Error al hacer el Push

### DISCUSSION THREAD

**Discussion by IsabelMD** [Taller-Abierto-de-Humanidades-Digitales/curso_python_extenso#6] [Dudas del curso]
Hola, Jairo. 

Espero se encuentre muy bien 

Me encuentro con algunas dificultades a la hora de hacer el Push, creo que es debido a que no cuento con los permisos necesarios para acceder al repositorio. Dejo por aquí el error que me arroja visual code: 

isabelmd@Isabels-MacBook-Pro curso_python_extenso % git push -u origin IsabelMD
remote: Permission to Taller-Abierto-de-Humanidades-Digitales/curso_python_extenso.git denied to IsabelMD.
fatal: unable to access 'https://github.com/Taller-Abierto-de-Humanidades-Digitales/curso_python_extenso.git/': The requested URL returned error: 403
isabelmd@Isabels-MacBook-Pro curso_python_extenso % 

Muchas gracias por su ayuda, quedo super atenta.

**Comment by jairomelo** [comment]
Hola @IsabelMD ,

Ya te asigné los permisos para poder escribir en el repositorio del curso. Debes aceptar la invitación a la "organización" desde el menú [your organizations](https://github.com/settings/organizations)

Si tienes problemas al hacer el `push` me vuelves a escribir :)

Muchos saludos,
